tests/test_core.py

```diff
@@ -91,9 +91,13 @@ def child():
     child()
 
 
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
Did you mean for this line to be here? Seems like this is what's screwing up the Windows tests.